### PR TITLE
Update: error handling

### PR DESCRIPTION
**Pull request by ghost**
Fix error handling

### CODE REVIEW

**Comment by arghvn** [comment]
Fix error handling

**Comment by ghost** [comment]
Definition of functions without input

**Comment by ghost** [comment]
no bug and no syntax error

File: spot_data_responses.go [hunk 1]
```diff
@@ -104,6 +104,36 @@ type SingleMarketStatistics struct {
 	Data TickerData
 }
 
+// AllMarketStatistics
```
Comment (arshamalh) [inline spot_data_responses.go:107]: These structs are wrong,
You should define Data structures like JSON responses.
I've sent you more details on Telegram.

**Pull request closed without merging**